heudiconv/convert.py

```diff
@@ -271,16 +270,23 @@ def convert(items, converter, scaninfo_suffix, custom_callable, with_prov,
                                      prefix + scaninfo_suffix)
 
                 if not op.exists(outname) or overwrite:
-                    tmpdir = tempdirs('dcm2niix')
 
+                    # first copy files select files
+                    item_dicoms_dir = tempdirs('DICOMs')
+                    for dcm in item_dicoms:
```

Comment:
Please add some log message (I would have added info level above the loop) about this copying happening since otherwise it would be just "busy doing something" for a while